eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
 
 <fg=cyan>Welcome to ${installName}!</fg=cyan>
 
-<options=bold>Quick dev/test install:</>
+<options=bold>Quick install to test in local dev environment:</>
 <comment>    $  export SYMFONY_ENV="dev"</comment>
 ${installCommandText}
 <comment>    $  php ${consoleDir}/console server:run</comment>
 
 Note:
-- Instructions above assume the CLI user you execute these commands with is the same one that extracted/installed the software.
-- The last command will give you the url to the frontend of the installation, add "/admin" to reach backend.
+- Instructions assume CLI user you execute commands with is the same one that extracted/installed the software.
```

Comment:
```suggestion
- The instructions assume the CLI user you execute commands with is the same one that extracted/installed the software.
```

Reply:
Thanks, be aware Im on purpose try to be short here as terminal has a tendency to have fixed width.

Unsure if it matters, but now you know why I try to stay within width limits

Reply:
@alongosz Do you think width matters here?

If so, maybe we can try to rephrase the 3 remaining sentences to keep them short.

Reply:
> @alongosz Do you think width matters here?
> 
> If so, maybe we can try to rephrase the 3 remaining sentences to keep them short.

I might not know something here, but I'd say - no it shouldn't matter much[1] nowadays. My imagination is that if we had an issue with a fixed width terminal, such terminal would be 80 characters tops, which was already too long before those changes in the diff.

The question is if there was an actual issue reported in the past? That would help us determining if we should be worried about it.

[1] We shouldn't make it too long due to readability anyway, but those changes for me are quite within reasonable limits.

Reply:
> The question is if there was an actual issue reported in the past? That would help us determining if we should be worried about it.

nope